pkg/controller/operators/catalog/operator.go

```diff
@@ -1731,6 +1732,28 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 
 				plan.Status.Plan[i].Status = status
 
+			case configMapKind:
+				var cfg corev1.ConfigMap
+				err := json.Unmarshal([]byte(step.Resource.Manifest), &cfg)
+				if err != nil {
+					return errorwrap.Wrapf(err, "error parsing step manifest: %s", step.Resource.Name)
+				}
```

Comment:
might be nice for `StepResource` to have an unmarshal method -- just something to think about for the future

Reply:
Agreed, they all use the same logic here